client/js/libs/jquery/inputhistory.js

```diff
@@ -54,7 +54,7 @@
 			case 38: // Up
 			case 40: // Down
 				// NOTICE: This is specific to The Lounge.
-				if (e.ctrlKey || e.metaKey) {
+				if (e.ctrlKey || e.metaKey || self.data("autocompleting")) {
```

Comment:
I'm not massively worried about this, because the library was built for shout originally anyway. I'm not sure of a better way of doing this, and we've modified it loads anyway.